src/sys/windows/tcp.rs

```diff
@@ -494,6 +518,9 @@ impl ListenerImp {
             State::Empty => {}
             _ => return
         }
+
+        me.iocp.unset_readiness(EventSet::readable());
```

Comment:
Oh wait scratch that, it looks like `defer` will end up calling `push_event` which ends up doing the right thing for level triggering, so this should be fine.